### PR TITLE
Add keywords to npm package

### PR DESCRIPTION
**Pull request by LeeWannacott**
I added a link to the repository.

### CODE REVIEW

**Comment by strager** [comment]
Did this somehow break npm installation? CI reports a failure: https://github.com/quick-lint/quick-lint-js/runs/1796769315

Looks good to me otherwise.

**Comment by LeeWannacott** [comment]
I don't think it breaks anything, I don't see why it would break anything? After the time I made this pull request I believe you were making changes to npm and specifically yarn; I don't remember the specifics. Maybe it was something to do with those changes?

It's the exact same tests that failed on my dark theme pull request maybe it's a problem on my end? because I don't have yarn installed?  However, I thought the testing is done in the cloud.

All three checks failing are for testing yarn install locally: ./node_modules/.bin/quick-lint-js: No such file or directory

**Comment by strager** [comment]
> I don't think it breaks anything, I don't see why it would break anything? After the time I made this pull request I believe you were making changes to npm and specifically yarn; I don't remember the specifics. Maybe it was something to do with those changes?

Perhaps. Can you try rebasing onto master to get any fixes?

> ..., maybe it's a problem on my end? because, I don't have yarn installed? However, I thought the testing is done in the cloud.

Testing is done on the cloud. The software you have installed on your machine won't affect the CI build which is failing.

File: dist/npm/package.json [hunk 1]
```diff
@@ -1,7 +1,28 @@
...
+    "feedback",
+    "javascript",
+    "linter",
+    "linting",
+    "strager",
+    "lint-quick-js",
+    "js-quick-lint",
+    "js-lint-quick",
+    "quick-js-lint",
+    "lint-js-quick"
```
Comment (strager) [inline dist/npm/package.json:24]: Nit: Alphabetically sort these, please. strageP

**Review by strager — APPROVED** [review]

File: dist/npm/package.json (hunk 1 above)
Comment (LeeWannacott) [reply dist/npm/package.json:24]: I don't think alphabetical sort makes sense in this instance. I ranked them on the relevance of what I thought would be most likely searched for, secondly on the descriptiveness of what quick-lint does.

**Pull request merged**